integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java

```diff
@@ -366,7 +368,10 @@ public TBase create() {
   public List<CarbonLocalInputSplit> getInputSplits2(CarbonTableCacheModel tableCacheModel,
                                                      Expression filters)  {
     List<CarbonLocalInputSplit> result = new ArrayList<>();
-
+    if(config.getUnsafeMemoryInMb() != null) {
+      CarbonProperties.getInstance().addProperty(CarbonCommonConstants.UNSAFE_WORKING_MEMORY_IN_MB,
```

Comment:
please add this configuration description to presto/README.md

Reply:
Done